src/Bridge/Symfony/DependencyInjection/Configuration.php

```diff
@@ -48,6 +48,10 @@ private function addFlashMessageSection(ArrayNodeDefinition $node): void
     {
         $node
             ->children()
+                ->enumNode('form_type')
+                    ->defaultValue('standard')
+                    ->values(['standard', 'horizontal'])
```

Comment:
Please add a call to `info()`

Reply:
👍 I added the original one, because I'm not sure what is this for... 🙃